src/main/java/nl/novi/techiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,44 @@
+package nl.novi.techiteasycontroller.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/televisions")
```

Comment:
Alle endpoints in deze controller beginnen nu met `localhost:8080/api/televisions`